.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: node_js
-node_js: 
+node_js:
   - "stable"
 services:
   - postgresql
+before_script:
+  - psql -c 'CREATE DATABASE users_test;' -U postgres
+  - npm i sequelize -g
+  - sequelize db:migrate
 after_success:
-  - npm test
```

Comment:
This command should be run in the script not here. It completely overlooks the presence of the latter

Reply:
@kodek-sleuth kindly learn how to review, code which is in a red line indicates it has been deleted and changed with a new piece of code.